.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
+---
+<!--
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete fields you cannot populate and all the comments from this template (including this one!).
+The resulting YAML blocks should contain valid YAML.
+Mandatory fields are marked with the word MANDATORY for your convenience.
+-->
+```yaml
+# This block is the boilerplate of a KSY spec file. It must be a valid KSY. It is meant to be ready to be copied as it is in order to make the task of the implementer easier.
+meta: # see https://doc.kaitai.io/ksy_reference.html#meta for more info
+  id: # insert an ID you feel right to assign to the format here. KS currently has no namespaces, so the id MUST be unique within all the repo and you should be sure there will be no collisions. If in doubt, add a prefix of an app name. MANDATORY.
+  title: # insert a human-readable name of the format here. MANDATORY.
+  application: # insert a human-readable name of the application for which this format is native here. If it is native for several applications, choose the ones you feel the most affiliated with the format. If the format is a widespread one, like PNG or WAV, just remove this field.
+  file-extension:
+    # insert here a YAML list of file extensions should be placed here.
+  xref: # check if the following sources contain info about the format you request. See https://doc.kaitai.io/user_guide.html#meta for more info
+    mime: # MIME type
+    rfc: # Link to RFC
+    din:
+      id: # DIN standard identifier
+    iso: # ISO standard identifier
+    forensicswiki: # Page name on https://forensicswiki.org/
+    justsolve: # ID on http://fileformats.archiveteam.org/wiki/
+    pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
+     - x-fmt/ 
+    wikidata: Q # ID on WikiData
+    fileformat: # Link to https://www.fileformat.info
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/    din:
+      id: # DIN standard identifier
+
+doc: |
+  # Describe your format briefly AND add links to the sources of sample files in the Web. MANDATORY
+
+doc-ref:
+  # A YAML array of URIs to information sources and/or their names goes here. MANDATORY. See https://doc.kaitai.io/ksy_reference.html#doc-ref for more info on sources which are not just an URI
+  # An effective strategy may be to search specifically for PDFs (filetype:pdf in Google) and source code on GitHub (links to source code files are especially valuable for understanding poorly documented parts) or directly in the repos of free software using the format.
+  # Prefer sources with a public-domain-like licenses over permissive ones, permissive ones over copyleft ones and free licenses over proprietary ones
+```
+
+```yaml
+# This block carries some machine-readable and machine-editable information which is not meant to be a part of the spec itself.
```

Comment:
I'm suggesting to omit this part at all.

Referencing an issue using normal GitHub mechanisms (i.e. adding a link somewhere in a text) is more than enough for practical purposes — you can just open an issue, count these mentions and see how many format ideas depend on this one. There is little to no value in having a special YAML format to mention these blockers. There's very vague distinction between "block" and "affect".

This creates tons of unnecessary complexity, *especially* at the beginning of new format creation. We'd want to encourage people to start these issues and chip in with references and knowledge about format, not turn the off with filling tons of forms of obscure nature, which they will most likely have no clue about at this point anyways.

Reply:
>We'd want to encourage people to start these issues and chip in with references and knowledge about format, not turn the off with filling tons of forms of obscure nature, which they will most likely have no clue about at this point anyways.

The stuff not marked as `MANDATORY` is there to notify the people "hey, this kind of info may be useful, we have a convention aboup where and in which format we put this, please add there that info if you have any and wanna fill in". If people are not ready to fill in the non-mandatory stuff, they can delete it. Unfortunately GH has no forms, maybe I should ask the suppkrt about them.